content/pages/directory.md

```diff
@@ -4,9 +4,9 @@ Summary: Information about the SpaceAPI directory
 SortOrder: 40
 
 The directory hosts a list of hackerspaces which implement the SpaceAPI. It is
-currently maintained by the [FIXME Hackerspace](https://fixme.ch/).
+currently hosted by the [FIXME Hackerspace](https://fixme.ch/).
```

Comment:
Is this still true? We now have https://directory.spaceapi.io/. Probably worth updating too.

Reply:
I compared the two endpoints and ours is still a bit buggy, see https://github.com/SpaceApi/website/issues/35.

Reply:
Good point :slightly_smiling_face: